drivers/spi/Kconfig.nrfx

```diff
@@ -14,6 +14,20 @@ menuconfig SPI_NRFX
 
 if SPI_NRFX
 
+if SOC_NRF52832
```

Comment:
Since there is only one option inside this `if`, it seems that using `depends on` in this option would be a better choice.

Reply:
Uhh... yes that does seem like a better choice. 😄 

_(Sorry, it's my first time writing `Kconfig` files and that must have slipped my mind...)_